Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -63,6 +63,36 @@ select new
             return Json(await equipment.ToListAsync());
         }
 
+        [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Equipment>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> SearchAttributes(string q)
+        {
+            var attributes = from ea in _context.EquipmentAttributes
+                    .Where(a => a.Equipment.Team.Slug == Team && a.Equipment.Active &&
+                                (EF.Functions.Like(a.Value, q.EfStartsWith())))
+                    .AsNoTracking()
+                select new
+                {
+                    ea.Equipment.Id
+                };
+            var ids = await attributes.Distinct().Select(a => a.Id).ToArrayAsync();
+
+            var equipment =
+                from eq in _context.Equipment
+                    .Where(x => x.Team.Slug == Team && x.Active && ids.Contains(x.Id))
+                    .Include(x => x.Attributes)
+                    .Include(x => x.Space).Include(x => x.Assignment)
+                    .OrderBy(x => x.Assignment != null).ThenBy(x => x.Name)
+                    .AsNoTracking()
+                select new
+                {
+                    equipment = eq,
+                    label = eq.Id + ". " + eq.Name + " " + eq.SerialNumber,

```

Comment:
is label something we use in other places? I don't remember ever seeing it before.  Is it more useful than just returning the equipmnet?

Reply:
I copied that from the search above:
https://github.com/ucdavis/Peaks/blob/master/Keas.Mvc/Controllers/Api/EquipmentController.cs#L47